src/main/java/it/pagopa/interop/probing/response/updater/client/EserviceClient.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.interop.probing.response.updater.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import it.pagopa.interop.probing.response.updater.dto.impl.ChangeResponseReceived;
+import it.pagopa.interop.probing.response.updater.exception.EserviceNotFoundException;
+
+@FeignClient(name = "eserviceClient",
+    url = "${api.operations.baseUrl}" + "${api.eservice.basePath}")
```

Comment:
`api.operations` and `api.eservice` seems two different services, but are actually the same.
Please rename the configs so that it is clear that the second is just one of the base path of the first